flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/batch/table/SortITCase.scala

```diff
@@ -57,14 +57,8 @@ class SortITCase(
       - x.productElement(0).asInstanceOf[Int] )
 
     val expected = sortExpectedly(tupleDataSetStrings)
-    val results = t.toDataSet[Row].mapPartition(rows => Seq(rows.toSeq)).collect()
-
-    val result = results
-      .filterNot(_.isEmpty)
-      .sortBy(_.head)(Ordering.by(f=> f.toString))
```

Comment:
We should change this to ` .sortBy(_.head)` and provide an implicit ordering for `Row`. 
When we changed `Row` to not extend `Product`, we should have added the implicit ordering for `Row` instead of sorting by String.

The same applies to the other tests in this class.